classes/lib/index.js

```diff
@@ -1,7 +1,12 @@
 const addon = require('../native/index.node');
+const assert = require("assert");
 
 const { User } = addon;
 
-new User(0, 'John', 'Doe', 'johndoe@gmail.com');
+const email = "johndoe@gmail.com";
+
+let user = new User(0, "John", "Doe", email);
+
+assert.strictEqual(user.email, email);
```

Comment:
This should be `user.get("email")`. The example is defining a function named `get` and not a JavaScript getter.